plugin/gv.vim

```diff
@@ -360,6 +371,13 @@ function! s:gv(bang, visual, line1, line2, args) abort
   endtry
 endfunction
 
+function! s:refresh() abort
+  let win = winsaveview()
+  tabclose
+  call s:gv(s:launch_opts.bang, s:launch_opts.visual, s:launch_opts.line1, s:launch_opts.line2, s:launch_opts.args)
+  call winrestview(win)
```

Comment:
If it only refreshes the parameters from the most-recent tab, instead of the current tab, it's not really useful. Sometimes I have different views open: `GV --since...`,  `GV HEAD^2..HEAD`, etc.

I'm not necessarily advocating for "current tab" functionality, but _not_ supporting it yet offering `r` is just confusing.